tests/sparsezoo/utils/test_onnx.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# noqa: F811
+# flake8: noqa
```

Comment:
why?

Reply:
Resolving this for now, happy to make this change in a future PR if needed